skill/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021, Mycroft AI Inc.
```

Comment:
I am not concerned either way, but it feels like the `skill` directory could be better as `date` at least for this Skill because the response and util modules are very date specific.

Reply:
I want this directory to be named `skill` for all skills.  The long-term goal will be to get all the skill's python code into this package, making it aptly named.